modules/service-fips.sh

```diff
@@ -121,16 +121,22 @@ _fips_enabled_check() {
     echo 0
 }
 
+_fips_check_installed() {
+    if ! _fips_check_packages_installed; then
+        return
```

Comment:
Shouldn't we return 1 here? What happens with such an empty return?

Reply:
This is the same as `return 0`.
The logic here is inverted, the function returns nonzero if packages are installed, as it's used as
`_fips_check_installed || error_exit service_already_enabled`